triqler/triqler.py

```diff
@@ -1,6 +1,13 @@
 from __future__ import print_function
 
 """triqler.triqler: provides entry point main()."""
+
+__version__ = "0.6.2"
+__copyright__ = '''Copyright (c) 2018-2020 Matthew The. All rights reserved.
+Written by Matthew The (matthew.the@scilifelab.se) in the
+School of Engineering Sciences in Chemistry, Biotechnology and Health at the 
+Royal Institute of Technology in Stockholm.'''
+
```

Comment:
Looks like you overwrote some of the code in a merge conflict. Please use the newer way of setting the version that you deleted below.